nevergrad/optimization/optimizerlib.py

```diff
@@ -1046,6 +1046,8 @@ class SplitOptimizer(base.Optimizer):
         number of optimizers
     num_vars: int or None
         number of variable per optimizer.
+    max_num_vars: int or None
+        maximum number of variables per optimizer.
```

Comment:
this may need clarification, what takes precedence? how does that work?

Reply:
Doc clarified.